core/mixins/_retina-bg-image.scss

```diff
@@ -17,11 +17,13 @@
  */
 
 
-@mixin retina-bg-image($image, $width, $height) {
-  @media (-o-min-device-pixel-ratio: 5/4),
-         (-webkit-min-device-pixel-ratio: 1.25),
-         (min-resolution: 120dpi) {
-           background-image: url($image);
-           background-size: $width $height;
+@mixin retina-bg-image($image, $width: auto, $height: auto, $resolution: $retina-resolution) {
+  $width: if($width, $width, auto);
+  $height: if($height, $height, auto);
+  $resolution: if(unitless($resolution), $resolution * 96dpi, $resolution);
+  
+  @media (min-resolution: $resolution) {
+    background-image: url($image);
+    background-size: $width $height;
   }
```

Comment:
> but I think you have to do that when working with Sass arguments? Which could also be the case for the 1st thing I mentioned above?

Yeah, this is an argument limitation. You have to pass arg 2 if you want to use arg 3. You can always just pass in `""' for something quick, but it would be easier for the developer just to calculate the `width` and have the `height` auto for them